src/controller_examples/zookeeper_controller/exec/zookeeper_lib/lib.rs

```diff
@@ -41,8 +77,8 @@ impl ExternalLibrary<ZKSupportInput, ZKSupportOutput> for ZKSupport {
     #[verifier(external)]
     fn process(input: ZKSupportInput) -> Option<ZKSupportOutput> {
         match input {
-            ZKSupportInput::ReconcileZKNode(zk)
-                => Option::Some(ZKSupportOutput::ReconcileZKNode(helper_funcs::reconcile_zk_node(&zk))),
+            ZKSupportInput::ReconcileZKNode(s1,s2,s3)
```

Comment:
Can you give them more descriptive names as @ZichengMa suggested then I will merge the PR